cfg/agentinfo/info.go

```diff
@@ -19,6 +19,10 @@ const versionFilename = "CWAGENT_VERSION"
 // We will fall back to a major version if no valid version file is found
 const fallbackVersion = "1"
 
+var isRunningAsRoot = func() bool {
+	return os.Getuid() == 0
```

Comment:
Should this account for the OS? I don't believe run_as_user is supported on windows hosts and `isRunningAsRoot` will always be false.